src/silo/database.cpp

```diff
@@ -448,6 +448,8 @@ void Database::preprocessing(
    const config::DatabaseConfig& database_config_
 ) {
    database_config = database_config_;
+   SPDLOG_DEBUG("preprocessing - using database config: {}", database_config);
```

Comment:
I think we should
* either write a custom log message not using the fmt lib so that this line doesn't pretend to log the full object
* or actually log the full object (adapting the fmt implementation to include all fields)